recipes/unitig-caller/meta.yaml

```diff
@@ -8,7 +8,7 @@ package:
 source:
   git_url: https://github.com/johnlees/{{ name|lower }}.git
```

Comment:
```suggestion
  url: https://github.com/johnlees/{{ name }}/archive/v{{ version }}.tar.gz
```

Reply:
And it turns out the `git_url` is required because the GitHub archive link does not include submodules.